src/PluginManager/Microsoft.Performance.Toolkit.PluginManager.Core/Packaging/Metadata/PluginMetadata.cs

```diff
@@ -0,0 +1,59 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+
+namespace Microsoft.Performance.Toolkit.PluginManager.Core.Packaging.Metadata
+{
+    /// <summary>
+    /// Represents the metadata of a plugin
+    /// </summary>
+    public sealed class PluginMetadata
+    {
+        /// <summary>
+        /// The group this plugin belongs to
+        /// </summary>
+        public string Group { get; set; }
```

Comment:
Can you elaborate on this? I'm not sure what Group means here

Reply:
Removed for now